src/main/java/com/openlattice/chronicle/controllers/legacy/ChronicleStudyController.java

```diff
@@ -64,7 +64,7 @@ public UUID enrollSource(
             @PathVariable( DATASOURCE_ID ) String datasourceId,
             @RequestBody Optional<Datasource> datasource ) {
 
-        return enrollmentManager.registerDatasource( null, studyId, participantId, datasourceId, datasource );
+        return enrollmentManager.registerDataSource( null, studyId, participantId, datasourceId, datasource );
```

Comment:
This naming change seems counterproductive since `datasource` is one word basically everywhere else in the codebase including the `Datasource` object. I would keep it as `registerDatasource` implying that its one word

Reply:
➕